packages/react/src/components/common/hocs/with-editor.tsx

```diff
@@ -0,0 +1,40 @@
+import {
+  ComponentType, useCallback, useMemo, useState,
+} from 'react';
+import { EditorProps } from '../../../internal/props';
+import { EditorContext } from '../contexts/editor-context';
+
+export function withEditor<T>(Component: ComponentType<EditorProps<T>>) {
+  function Editor<V extends T>(props: EditorProps<V>) {
+    const [editorValue, setEditorValue] = useState(props.value || props.defaultValue);
+    const [editorErrors, setEditorErrors] = useState<string[]>();
+
+    const setEditorErrorsCallback = useCallback((errors: string[]) => {
+      setEditorErrors(errors);
+    }, [setEditorErrors]);
+
+    const editorContextValue = useMemo(
+      () => ({ editorName: props.name || '', editorValue, setEditorErrors: setEditorErrorsCallback }), [editorValue, props.name],
```

Comment:
Add setEditorErrorsCallback into the deps

Reply:
fixed